examples/nodejs/aws/eventbridge/webapp/src/components/InfoModal.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from "react";
+import { Help } from "../svgs/help";
+
+type InfoModalProps = {
+  isVisible: boolean;
+  onClose: () => void;
+};
+
+const appUsageInfo = `
+<strong>Application Overview:</strong><br>
+The application showcases a write-through cache pattern implementation for DynamoDB utilizing DynamoDB Streams, AWS EventBridge, and Momento. It enables users to perform create, update, and delete operations on items within a DynamoDB table, with real-time reflection of changes in the cache (<span class="italic">"momento-eventbridge-cache"</span>) and topic (<span class="italic">"momento-eventbridge-topic"</span>).<br><br>
+
+<strong>How to Use the App:</strong><br><br>
+<strong>Prerequisites:</strong><br>
+Before interacting with the application, ensure the existence of the cache <span class="italic">"momento-eventbridge-cache"</span>. If the cache does not exist, it can be created via <a class="text-blue-500" href="https://console.gomomento.com" target="_blank" rel="noopener noreferrer">Momento Console</a>.<br><br>
```

Comment:
`ensure the existence` -> `you'll need to create a cache called ... in the [Momento Console](...). If you haven't done that already, please take a moment to do it now!`